src/js/components/NameValueList/stories/data.js

```diff
@@ -19,7 +19,7 @@ export const metricData = {
 };
 
 export const statusData = {
-  Power: 'Ok',
+  Power: 'Management for campus, branch, remote, and data center network',
```

Comment:
In the spirit of keeping our stories real, this long string doesn't seem to be appropriate for "Power". Perhaps we can just add another pair?

Reply:
yes I can add another pair that is more of a "real" story.. I was not really wanting to change the story just wanted to  put context behind why the `icon` does not align with `text`